git_hg_sync/__main__.py

```diff
@@ -88,7 +88,7 @@ def main() -> None:
     sentry_config = config.sentry
     if sentry_config and sentry_config.sentry_dsn:
         logger.info(f"Sentry DSN: {sentry_config.sentry_dsn}")
-        sentry_sdk.init(sentry_config.sentry_dsn)
+        sentry_sdk.init(sentry_config.sentry_dsn, max_value_length=4096)
```

Comment:
Why the max here?

Reply:
The exception captures the stdout/stderr from the process. With hooks, the HgMO output can be quite verbose, particularly in case of errors, and go beyond the 1024 bytes default.